src/components/search/search.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Search = ({
+    onZipCodeChange
+}) => {
+    return (
+        <div className="search-form">
+            <form className="js-search-form">
+                <fieldset className="js-search-fieldset">
+                    <div className="search">
+                        <div className="search__input">
+                            <label htmlFor="searchzipcode" className="label search-label">Zip Code</label>
+                            <input className="input js-search" id="searchzipcode" name="searchzipcode" type="search" placeholder="Search Zip Code" value="" pattern="[0-9]{5}" maxLength="5" />
```

Comment:
[fix] You're throwing some console errors here. Let's not worry about this yet -- it has to do with controlled vs. uncontrolled form inputs, but we can address this when you start to work on forms in React.